chukwuemekachm/M-Tracker-React-Client
# #160296082 Completes user create request feature

#### What does this PR do?
- This PR completes user create request feature
#### Description of Task to be completed?
- Add CreateRequest.jsx
- Modify SideNav.jsx
- Modify Dashboard.jsx
- Unit test CreateRequest component
- Modify existing tests
- Update snapshots
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser should be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- You will be redirected to the dashboard
- Click the `Create Request` button and fill the form that pops up
- Click the `Create Request` button below the form to create a new request
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160296082](https://www.pivotaltracker.com/story/show/160296082)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-01 at 3 09 21 pm" src="https://user-images.githubusercontent.com/33798252/46293978-adfe0800-c58c-11e8-90a1-0191cbde9263.png">


<img width="1440" alt="screen shot 2018-10-01 at 3 09 33 pm" src="https://user-images.githubusercontent.com/33798252/46293979-ae969e80-c58c-11e8-8e3d-cd51afb3b1db.png">

#### Questions:
- None
